.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -25,5 +25,5 @@ As an open source project - it may take some time for your issue to be addressed
 
 ### Versions
 
-<!-- Valora version, Opperating System - iOS or Android, Devices e.g. LG K7i(6.0), and any additional information as needed -->
+<!-- Valora version, operating system - iOS or Android, devices e.g. LG K7i(6.0), and any additional information as needed -->
```

Comment:
I wonder if we can use something more familiar to users instead of `LG K7i(6.0)`. Perhaps `iPhone 12 Max` or `Pixel 5` or something?

Reply:
@tarikbellamine I've updated the device text in commit [d4fae6d](https://github.com/celo-org/wallet/pull/316/commits/d4fae6d9639ce2d15cf950f59a68b334c46a7c36).